zuul.d/jobs.yaml

```diff
@@ -2139,3 +2139,19 @@
       - OS:ubuntu-xenial
       - Arch:x86_64
       - BuildType:Integration test
+
+# hbase arm build periodic job
+- job:
+    name: hbase-build-arm64
+    parent: init-test
+    description: |
+      Hbase build in openlab arm instance.
+    run: playbooks/hbase-arm-build/run.yaml
+    nodeset: ubuntu-xenial-arm64
+    tags:
+      - Category:BigData
+      - Project:apache/hbase
+      - Application:Hbase@master
+      - OS:ubuntu-xenial
+      - Arch:aarch64
+      - BuildType:Integration test
```

Comment:
Not sure if  we have any other BuildType? Seems this job only does `compile` and  `package` action, no test at all. Does `Integration test` is suitable here? Or you'll add the `test` action later?

Reply:
Good catch, according to @huangtianhua 's suggestion, the following types are options
1. Integration test
2. Unit test
3. Compile

Reply:
Thanks. It's now clear.